### DISCUSSION TITLE
While deploying my web app for technical onboarding I just stucked in deploying my web app with azure in vscode

### DISCUSSION THREAD

**Discussion by Buraquescode** [microsoft/SATechnicalOnboarding#78] [Questions]
While deploying my web app for technical onboarding I just stucked in deploying my web app with azure in vscode? any solution?

**Comment by AnthonyByansi** [comment]
Hi @Buraquescode, please note that the Azure VScode extensions aren't that supposed optimumly, You should consider deploying that manually in the Azure portal  instead of relying on the Azure Vscode extension. Here is the [Documentation](https://learn.microsoft.com/en-us/azure/static-web-apps/getting-started?tabs=vanilla-javascript) you can follow to for a smooth setup process.
